worker/fbs/transport.fbs

```diff
@@ -13,11 +13,17 @@ enum Protocol: uint8 {
     TCP
 }
 
+enum SocketFlag: uint8 {
+    IPV6ONLY = 1,
+    UDP_REUSEPORT = 2
+}
+
 table ListenInfo {
-    protocol: FBS.Transport.Protocol = UDP;
+    protocol: Protocol = UDP;
     ip: string (required);
     announced_ip: string;
     port: uint16 = 0;
+    flags: uint8 = 0;
```

Comment:
IMO for readability, rather than this being an already formed integer with the contained flags, it could be an array of `SocketFlags`:

```txt
flags: [SocketFlag];
```

The corresponding integer would then only be formed in C++.

Reply:
Let's not make it less performant, please. We don't want to iterate all values in worker side and check each bit against supported libuv flags. This is a perfect and optimized usage of bit checking.

Reply:
Can't you compress it into bitfield on the worker and leave it well typed in flatbuffers?

Reply:
Or even:

```txt
table SocketFlags {
  ipv6_only: bool = false;
  udp_reuse_port: bool = false;
}
```

Then `ListenInfo` would have:

```txt
  flags: SocketFlags;
```

I'm just commenting this because it looked strange to me forming a flag integer from TS rather than encoding the corresponding meaningful values over flatbuffers.

Reply:
> Let's not make it less performant, please

No single iteration with previous proposal. Not in TS/Rust, not in Worker, plus removed the unneeded vector usage for a well known type.

Reply:
Will do